.github/workflows/docker-publish.yml

```diff
@@ -16,7 +16,7 @@ permissions:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
+  IMAGE_NAME: ${{ lower(github.repository) }}
```

Comment:
`lower()` is not a function supposed in https://docs.github.com/en/actions/reference/evaluate-expressions-in-workflows-and-actions#functions